oauthenticator/__init__.py

```diff
@@ -1,6 +1,7 @@
 # include github, bitbucket, google here for backward-compatibility
 # don't add new oauthenticators here.
 from .oauth2 import *
+from .generic import *
```

Comment:
See the comment above for 'don't add new things here'. Authenticators should be imported directly from their submodule, so that we don't import all the unused Authenticators as this package grows.